apps/book/admin.py

```diff
@@ -0,0 +1,67 @@
+from django.contrib import admin
+from apps.book.models import Tag, Category, Author, Book
+from modeltranslation.admin import TranslationAdmin
+
+
+class TagAdmin(TranslationAdmin):
+    list_display = ['id', 'name']
+    list_display_links = ['id', 'name']
+    search_fields = ['id', 'name', ]
+
+
+class CategoryAdmin(TranslationAdmin):
+    list_display = ['id', 'name', 'parent_category']
+    list_display_links = ['id', 'name']
+    search_fields = ['id', 'name', ]
+
+
+class AuthorAdmin(TranslationAdmin):
+    list_display = ['id', 'name']
+    list_display_links = ['id', 'name']
+    search_fields = ['id', 'name', ]
+
+
+class BookAdmin(TranslationAdmin):
+    list_display = ['id', 'title', 'book_authors', 'book_tags', 'book_categories', 'publisher']
+    autocomplete_fields = ('categories', 'tags', 'authors', 'publisher')
+    search_fields = [
+        'id', 'title', 'authors__name', 'categories__name', 'publisher__publisher_name', 'tags__name',
```

Comment:
`publisher__publisher_name` this seems inconsistent, may be we should just have `publisher__name`